@Kenny2github Getting close! Looks like you are just failing the linter:
```bash
 112:58  error  Expected line break before `.includes`  newline-per-chained-call
```

I would recommend breaking up the line a bit to make it more readable / pass lint.

--- a/src/blocks/scratch3_operators.js
+++ b/src/blocks/scratch3_operators.js
@@ -109,8 +109,10 @@ class Scratch3OperatorsBlocks {
     }
     
     contains (args) {
-        return Cast.toString(args.STRING1).toLowerCase()
-            .includes(Cast.toString(args.STRING2).toLowerCase());
+        const format = function (string) {
+            return Cast.toString(string).toLowerCase()
+        }
+        return format(args.STRING1).includes(format(args.STRING2))
     }
 
     mod (args) {

--- a/src/blocks/scratch3_operators.js
+++ b/src/blocks/scratch3_operators.js
@@ -110,9 +110,9 @@ class Scratch3OperatorsBlocks {
     
     contains (args) {
         const format = function (string) {
-            return Cast.toString(string).toLowerCase()
-        }
-        return format(args.STRING1).includes(format(args.STRING2))
+            return Cast.toString(string).toLowerCase();
+        };
+        return format(args.STRING1).includes(format(args.STRING2));
     }
 
     mod (args) {

--- a/src/blocks/scratch3_operators.js
+++ b/src/blocks/scratch3_operators.js
@@ -109,7 +109,8 @@ class Scratch3OperatorsBlocks {
     }
     
     contains (args) {
-        return Cast.toString(args.STRING1).toLowerCase().includes(Cast.toString(args.STRING2).toLowerCase());
+        return Cast.toString(args.STRING1).toLowerCase()
+            .includes(Cast.toString(args.STRING2).toLowerCase());
     }
 
     mod (args) {

